src/llama_stack_client/lib/agents/agent.py

```diff
@@ -51,7 +57,10 @@ async def create_turn(
         )
         turn = None
         for chunk in response:
-            if chunk.event.payload.event_type != "turn_complete":
+            if hasattr(chunk, "error"):
```

Comment:
What is the chunk's value that will be returned if an error happens?

Reply:
yesh seems like we return either
- a chunk
- a message (for tool call) ! 
- and now, an error dict!